src/d3a/models/strategy/settlement_strategies/__init__.py

```diff
@@ -0,0 +1,142 @@
+from d3a_interface.constants_limits import ConstSettings
+from pendulum import duration
```

Comment:
1. Will rename this to settlement/strategy.py
2. Not really. d3a_interface should precede pendulum since they are both third-party imports, and d3a_interface precedes pendulum in alphanumeric order
3. Already added from previous comments